guides/Codable.md

```diff
@@ -0,0 +1,102 @@
+## I want to make my types conform to `Encodable` or `Decodable`
+
+### [Encodable Template](https://github.com/krzysztofzablocki/Sourcery/blob/master/Templates/Templates/Encodable.stencil)
+### [Decodable Template](https://github.com/krzysztofzablocki/Sourcery/blob/master/Templates/Templates/Encodable.stencil)
+
+#### Available annotations
+
+
+* `skipCoding` - skips decoding and encoding
+* `codingKey` - The coding key to use, this is optional and if no `codingKey` annotation is found the variable name is used.
```

Comment:
Can be useful to have also `decodingKey` and `encodingKey` in case they happen to be different